### DISCUSSION TITLE
Remove: virus theft github account text and user

### DISCUSSION THREAD

**Discussion by ghost** [community/community#122673] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### factual evidence
```
https://github.com/z-hao-wang/go-ethereum-qs/pull/1#issuecomment-2080389904
https://github.com/search?q=Blastoff.zone%3A+Earn+1+ETH&type=issues
https://github.com/search?q=Blastoff.zone%3A+Earn+1+ETH&type=pullrequests
https://github.com/search?q=Blastoff.zone%3A+Earn+1+ETH+%2B+%24500+Bonus%21&type=issues
https://github.com/search?q=Blastoff.zone%3A+Earn+1+ETH+%2B+%24500+Bonus%21&type=pullrequests

https://github.com/S0FTWxR3/github-slideshow/issues/2
https://github.com/search?q=As+part+of+this+position%2C+you+will+be+offered+a+competitive+salary+of+%24180%2C000+per+year%2C+along+with+other+attractive+benefits%2C+including&type=issues
https://github.com/search?q=As+part+of+this+position%2C+you+will+be+offered+a+competitive+salary+of+%24180%2C000+per+year%2C+along+with+other+attractive+benefits%2C+including&type=pullrequests&p=2
https://github.com/aisk/browsercookies/issues/14
https://github.com/cdrn/AircraftAssignment/issues/1
https://github.com/ironsoul0/betting-platform/pull/1
https://github.com/iggyiccy/decentralised-game-trading/pull/12
https://github.com/meet-cjli/CTRL/issues/3
https://github.com/VictorHDS/CybersecurityChallengePhishing/issues/1
https://github.com/keeganmccallum/devgpt-releases/pull/1
```

### criminal user
```
https://github.com/Modelizer
https://github.com/thangtq-pionero
https://github.com/mohit-deepgrid
https://github.com/VictorHDS
https://github.com/vissequ
https://github.com/iggyiccy
https://github.com/Ripon9898
https://github.com/willianfcamara
https://github.com/smcgive
https://github.com/ripperr
https://github.com/pavana21
https://github.com/appsplash99
https://github.com/ironsoul0
https://github.com/BhavyClover
https://github.com/Arpitkandwal
https://github.com/leojin
https://github.com/cheransengodan-sensiple
```

### Criminal text links

![](https://github.zendesk.com/attachments/token/CwQDADbis1PP2zNfptiRBmSy3.JPG)
![2.JPG](https://github.zendesk.com/attachments/token/Qq1QC0m2r4bQryiOxogf5DZYY.JPG)
![3.JPG](https://github.zendesk.com/attachments/token/KbD4Kya8qKRV2AYloPppf4vln.JPG)
![4.JPG](https://github.zendesk.com/attachments/token/NVBKaGwEYtHETMSnaikW6emAC.JPG)
![5.JPG](https://github.zendesk.com/attachments/token/MyxXrAEc3EDuWEVPwNBi5TsTm.JPG)

![20220412124704_1.jpg](https://github.zendesk.com/attachments/token/lqKkrOLpxQCi8nIK1ecupO2SZ/?name=20220412124704_1.jpg)

**Comment by ebndev** [comment]
Hi @Zig-Rust-Odin 👋🏻

The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
